BUILD.md

```diff
@@ -364,6 +365,29 @@ cd $GOPATH/src/github.com/linkerd/linkerd2
 bin/update-codegen.sh
 ```
 
+## Helm chart
+
+The main Linkerd2 chart is located under [`charts/linkerd2`](charts/linkerd2).
+It depends on the chart under [`charts/partials`](charts/partials). The chart
+under [`charts/patch`](charts/patch) is only used internally for the sidecar
+proxy injection.
+
+The charts are installed and managed with the Helm CLI, but we encourage you to
```

Comment:
I think we should just tell contributors to use `bin/helm`. So something like,

_During development, please use the `bin/helm` wrapper script to invoke the Helm commands. For example,_
```
bin/helm install charts/linkerd2
```
_This ensures that you use the same Helm version as that of the Linkerd CI system._